manifests/m/Microsoft/OpenSSH/Beta/9.2.0.0/Microsoft.OpenSSH.Beta.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with Komac v1.0.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.OpenSSH.Beta
+PackageVersion: 9.2.0.0
+InstallerLocale: en-US
+InstallerType: msi
+Scope: machine
+AppsAndFeaturesEntries:
+- DisplayName: OpenSSH
+ReleaseDate: 2023-02-16

```

Comment:
this is the drafted date I think, the release date should be 2023-02-21

Reply:
It probably picked the ReleaseDate from the asset dates in the releases.

Reply:
- https://github.com/microsoft/winget-pkgs/pull/97520